packages/documentation/src/pages/getting-started/common-tasks/new-application.mdx

```diff
@@ -113,16 +126,16 @@ And that's it! If you fill out the routes.jsx file with pages that exist, you sh
 
 ## Going to production
 
-The above steps will only enable your application in our staging and dev environments. In order to enable it in production, you'll have to create a content page for your application. In dev/staging, our Metalsmith pipeline will create a temporary one.
-
-In `vagov-content`, you can add a page with the following frontmatter:
-
-```md
----
-title: My new application 
-entryname: my-new-application
-layout: page-react.html
----
-```
-
-The most important thing to note is that the `entryname` property should match the `entryName` property in your manifest file and that the page should be in the location that matches the `rootUrl` path. In this case, that would be `pages/my-new-application.md` or `pages/my-new-application/index.md` in `vagov-content`.
+The above steps will only enable your application in our staging and dev environments. In order to enable it in production, you'll have to remove the feature flag added into your application's `manifest.json`.
+
+<pre>
+"appName": "My new application",
+"entryFile": "./my-new-application-entry.jsx",
+"entryName": "my-new-application",
+"rootUrl": "/my-new-application",
+"template": {
+  "title": "My new application",
+  "layout": "page-react.html",
+  <del>"vagovprod": false</del>
+}
+</pre>
```

Comment:
I wanted a strikethrough for the `vagovprod` part. Any better ideas?

Reply:
just `~~"vagovprod": false~~` but i'm not sure if they work:

~~"vagovprod": false~~